### DISCUSSION TITLE
Using Frigate for motion activated lights?

### DISCUSSION THREAD

**Discussion by sinewave** [blakeblackshear/frigate#3158] [Ask A Question]
I’ve currently been playing around with using my interior cameras as motion activated lights which has been working really well especially with presence detection!!

The only big issue I have that's preventing me from ditching my current motion sensors is detecting motion at night. I'm using Unifi G3 micros indoors and no matter what I've tried Frigate can't seem to detect **any** motion using IR when it's completely dark.

I tried switching from using Person detection to checking for > 0 "detection_fps" but that didn’t help either as it doesn’t change either. When viewing the camera in debug mode it is quite apparent that there is motion happening however no motion boxes are shown and nothing is detected unless I get 2-3 ft away from the camera. 

I also have some Unifi GP Pro and G3 Bullet outside and they all suffer from this issue.. almost nothing is detected at night.

Has anyone had any luck getting something like this to work - any settings I can play with?

P.S I'm using CPU detection as I'm still waiting on my Coral TPU like everyone else 😞

**Comment by NickM-27** [comment]
What version of Frigate are you using? There is a setting `improve_contrast` in 0.10.1 which will help detect motion in these situations

**Reply by sinewave** [reply]
I'm on 0.10.1 - thx i'll check it out!
